src/Features/Core/Portable/InheritanceMargin/AbstractInheritanceMarginService_Helpers.cs

```diff
@@ -38,26 +38,65 @@ internal abstract partial class AbstractInheritanceMarginService
                 SymbolDisplayMiscellaneousOptions.UseErrorTypeSymbolName |
                 SymbolDisplayMiscellaneousOptions.IncludeNullableReferenceTypeModifier);
 
-        public async ValueTask<ImmutableArray<InheritanceMarginItem>> GetInheritanceMemberItemAsync(
-            Project project,
-            Document? documentForGlobalImports,
+        public async ValueTask<ImmutableArray<InheritanceMarginItem>> GetGlobalImportItemsAsync(
+            Document document,
             TextSpan spanToSearch,
+            bool frozenPartialSemantics,
+            CancellationToken cancellationToken)
+        {
+            var solution = document.Project.Solution;
+            var remoteClient = await RemoteHostClient.TryGetClientAsync(solution.Workspace.Services, cancellationToken).ConfigureAwait(false);
+            if (remoteClient != null)
+            {
+                // Also, make it clear to the remote side that they should be using frozen semantics, just like we are.
+                // we want results quickly, without waiting for the entire SG pass to go.  The user will still get
```

Comment:
`entire SG`
Sorry, could you explain what's 'SG'?

Reply:
I think I know what it is after reading the following code, is it source generator? Could you inline the full name in the comments?

Reply:
Yup.  It's SourceGenerator.  i can definitely expand that in the comment :)

Reply:
done.